### PR TITLE
Reinforced tables require welding to construct/deconstruct

### PR DESCRIPTION
**Pull request by ArtisticRoomba**
## About the PR
Reinforced tables now require welding to construct and deconstruct.

## Why / Balance
This is primarily intended to combat this issue raised in #33871.

Chemists often just wrench their own table or someone else's table to steal plasteel for plasma.

I think that the 10u vial solution, while solving the problem, kinda distracts from the point of the game, to be a social and roleplaying game. Giving a 10u vial bypasses one of the few reasons regular chemists leave their department.

Now, chemists will have to talk to departments to get their plasma.

Welding is required for both constructing AND deconstructing to promote a neutral time and resource balance if someone wants to spam reinforced tables for... whatever reason.

## Technical details
Just added 3s doafter welding steps in the construction graph.

## Media

https://github.com/user-attachments/assets/bcc226a1-3db8-4a51-87ef-0a44087716ff

## Requirements

- [x] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [x] I have added media to this PR or it does not require an ingame showcase.

**Changelog**
:cl:

- tweak: Reinforced tables now require welding to construct and deconstruct.

### CODE REVIEW

**Comment by PJB3005** [comment]
This is how it is in SS13 so :+1:

File: Resources/Prototypes/Recipes/Construction/Graphs/furniture/tables.yml [hunk 1]
```diff
@@ -145,6 +147,8 @@
           steps:
             - tool: Anchoring
               doAfter: 1
+            - tool: Welding
+              doAfter: 3
```
Comment (PJB3005) [inline Resources/Prototypes/Recipes/Construction/Graphs/furniture/tables.yml:151]: I'd put welding *before* the anchoring step.

**Review by PJB3005 — APPROVED** [review]

**Review by Partmedia — APPROVED** [review]
Looks good to me. Minor nitpick: in the future consider reviewing the diff in "Files changed" to minimize unrelated changes outside of the part that's actually being changed.

**Comment by ArtisticRoomba** [comment]
yeah... I changed a file and then reverted the change, leaving all of those tasty whitespace changes... poor Rider loves inflicting formatting on everything it touches

**Comment by CookieMasterT** [comment]
I guess chemists are guaranteed to have oculine made every shift now so that's a win :)

**Pull request merged**